src/assets/stylesheets/main/layout/_nav.scss

```diff
@@ -113,8 +113,8 @@
         0 #{$margin-height} var(--md-default-bg-color);
     }
 
-    // Navigation link in blurred state
-    &[data-md-state="blur"]:not(&--in-viewport):not(&--active) {
+    // Navigation link that was passed
+    &--passed {
```

Comment:
Not sure if this is correct --- will have to test after the other issue I mentioned related to `--passed` is fixed.

Reply:
I allowed this because I noticed the change in [src/assets/javascripts/components/toc](https://github.com/jbms/sphinx-immaterial/pull/171/files#diff-ed6ae506600ba3c7de17fba6b64bc6a7f67a571894f3f73ce06ea04c63cb6b66). Are you specifically concerned about the `:not(&--in-viewport)`?

I didn't allow the change to the `toc.follow` feature (detailed in the patch_info.diff located in OP for this PR).

Reply:
Yeah, I'm not sure if the other selectors can be removed or if they are still required --- will need to test once you have pushed out a fix for the other `--passed` change.

Reply:
pushed that `--passed` change.

Reply:
If you've approved, then I'm guessing this can be resolved?

Reply:
yes, I verified that it works correctly